Gruntfile.js

```diff
@@ -92,6 +92,7 @@ module.exports = function(grunt) {
   // on each recompile
   var updateTsFiles = function() {
     tsFiles = grunt.file.read("src/reference.ts")
+      .replace(/\r\n/g, "\n")
```

Comment:
You´re right, it won´t help and the problem then runs deeper, becuase all the typescript compiles and the concats are going to use the windows linefeed, and neither git nor visual studio can affect that.

But another possibility could be to use
grunt.util.linefeed
instead of the literal \n

```javascript
   var updateTsFiles = function() {
     tsFiles = grunt.file.read("src/reference.ts")
       .split(grunt.util.linefeed)
```
If this were done everywhere, .gitattributes would not be needed?